src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -354,8 +389,67 @@ public JSONObject getDebugState()
     public abstract void addRtpExtension(RtpExtension rtpExtension);
 
     /**
+     * Sets the map of endpoint id to {@link VideoConstraints} that contains the
```

Comment:
Can you add something like "the video constraints for the streams that this endpoint wishes to receive"?